webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -335,6 +335,10 @@ public List<SnapshotInfo> findSnapshots(final String repositoryName, final Long
     final GetSnapshotResponse response;
     try {
       response = esClient.snapshot().get(snapshotsStatusRequest);
+      if (response.snapshots().isEmpty()) {
+        throw new ResourceNotFoundException(
```

Comment:
:thought_balloon: I'm not keen on artificially throwing an exception, and would rather that the caller handles the case where there is nothing found. But I don't know enough about the use case, honestly, and backups are not performance sensitive, so this is probably OK.